content/kotlin/concepts/strings/terms/endsWIth/endsWith.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.endsWith()'

```

Comment:
The path has an error. The folder name is "endsWIth" but it should be "endsWith"